src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentManagerExtension.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.ContentManagement
+{
+    public static class ContentManagerExtension
+    {
+        public static async Task UpdateAndCreateAsync(this IContentManager contentManager, ContentItem contentItem, VersionOptions options)
```

Comment:
@jtkech suggestion would be to make this method use a lambda param so that we can execute things in between UpdateAsync and CreateAsync. But for now it fixes all the issues we had so it's something that we could do later on.

Reply:
As you want ;) ATM i don't see what we may want to do between the two